frontend/pages/medical-recovery-centre/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+
+const MedicalRecoveryCentre = () => {
+  return (
+    <>
+      {style}
+      <div className="container">
+        <h1>Medical Recovery Centre</h1>
+        <TheCottage />
+        <TierneyHouse />
+        <TBAPerth />
+      </div>
+    </>
+  )
+}
+
+const TheCottage = () => {
+  return (
+    <>
+      <h2>The Cottage</h2>
+      <blockquote>
+        The Cottage is a supportive, home like environment where Complex Care
+        Services and Hospital in the Home (HITH) services are provided to people
+        who are homeless or at risk of homelessness.
+        <br />
+        <br />
+        The aim is to provide holistic, recuperative care to clients with a
+        nursing need, as an alternative to staying in hospital.
+        <br />
+        <br />
+        The Cottage focuses on building rapport and trust between clients and
+        staff, enabling staff to establish a safe environment and platform from
+        which they can address clients’ health issues.
+      </blockquote>
+      <div className="row flex-container">
+        <div className="col s12 m3 card-margin">
+          <a
+            href="https://drive.google.com/open?id=1bNTOFku6OnhaZta7gEB9BE9ZQhGzcV3X"
+            target="_blank"
+          >
+            <div class="card ">
+              <div class="card-image">
+                <img src="/images/the-cottage-infographic-preview.png" />
+              </div>
+              <div className="card-action">The Cottage Infographic</div>
+            </div>
+          </a>
+        </div>
+        <div className="col s12 m3 card-margin">
+          <a
+            href="https://drive.google.com/open?id=1VhauhRFC6ftOIkMhGsXw6QUjGPOTyJQA"
+            target="_blank"
+          >
+            <div class="card">
+              <div class="card-image">
+                <img src="/images/the-cottage-gazey-preview.png" />
+              </div>
+              <div className="card-action">
+                The Cottage: Providing Medical Respite Care (Gazey et al 2018)
+              </div>
+            </div>
```

Comment:
this one, I think I created a component called `LinkCard` and `LinkCards` where it takes an object that looks like this

```js
<LinkCards
        cardDetails={[
          {
            alt: '50 Lifes 50 Homes logo',
            src: '/images/Dual-Diagnosis-Services/CHOPS_Infographic_1.png',
            link: '/research_papers/SVHM_-_CHOPS_Infographic.pdf',
            reportTitle: 'CHOPS Infographic'
          }
        ]}
      ></LinkCards>
```

I used this to refactor the section that I made, also Han's work, and I think it makes it easier to just insert download cards

Reply:
Whoops~ Didn't notice that :P These shared components are also allow having a common style, one place to modify them all, and prevent duplicating work thus reducing the amount of work other developers have to do! Good work!